### PR TITLE
Adding local storage results in dashboard & fixed dosage color toggle

### PR DESCRIPTION
**Pull request by kristina-voit**
Please refer to #22 and #23 

- If a new card is created in appointment/medicine, it will be displayed on the dashboard as well
- The default color of "0" in dosage is now green

NOT working yet: 
- To-dos are not stored in local storage
- Creating new cards will render all cards in one card
- Graph is still only hard-coded

### CODE REVIEW

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Kristina, nice features, I just left some small remarks.
All in all I saw that you have several components that end with "short". I think it would be nice to refactor that in a later step and try to use the same component by controlling the "size" via a prop?

File: src/components/Events/NewEventShort.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+`;
+
+const FullCardContent = styled.div`
+  text-align: left;
+  margin: 1rem;
+`;
+
+const CardTitle = styled.p`
+  font-weight: 600;
+  margin: 12px 0px 0px 0px;
```
[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Events/NewEventShort.jsx:51]: Concerning the refactoring: Yes, will do! Added this to my project board as a task for the upcoming days!

[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Events/NewEventShort.jsx:51]: Concerning the 0px: changed in all files!

File: src/components/Medicine/Dosage.jsx [hunk 2]
```diff
@@ -9,6 +9,13 @@ export default function Dosage(props) {
         {props.einnahmeMenge}
       </Description>
     );
+  } else if (
+    props.einnahmeMenge == 0) {
+      return (
+        <Description color={!color}>
```
[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Medicine/Dosage.jsx:15]: Changed it to isColored!

File: src/components/Medicine/DosageShort.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      );
+  } else {
+    return (
+      <Description onClick={() => setColor(color)} color={color}>
+        {product.einnahmeMenge}
+      </Description>
+    );
+  }
+}
+const Description = styled.p`
```
[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Medicine/DosageShort.jsx:27]: No, I will reuse the Dosage-component! Task is on my project-board.

File: src/components/Medicine/NewMedicineShort.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import { useState } from "react";
+import { loadFromLocal } from "../../lib/localStorage";
+import Morning from "../../images/sunrise.svg";
+import Noon from "../../images/noon.svg";
+import Evening from "../../images/sunset.svg";
+import Night from "../../images/night.svg";
+import DosageShort from "../Medicine/DosageShort"
+
+
+function NewProduct() {
```
[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Medicine/NewMedicineShort.jsx:11]: This is indeed a very good question :-) I will check my whole project for naming issues!

File: src/components/Medicine/NewMedicineShort.jsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  return (
+    <div>
+      <Card>
+        {products.map((product, index) => (
+          <article>
+         
+            <CardTitle>{product.name}</CardTitle>
+
+            <FullCardContent>
+
```
[the remark this answers is not in the document]
Comment (kristina-voit) [reply src/components/Medicine/NewMedicineShort.jsx:28]: Done!

**Pull request merged**